ginga/Control.py

```diff
@@ -1419,6 +1479,52 @@ def _add_image_update(self, image, info):
             if channel != self:
                 self.fv.change_channel(self.name)
 
+    def image_data_modified(self, image, reset=False):
+        """Call this when data in buffer is modified.
+
+        Header metadata, ``MODIFIED``, will be updated with UTC timestamp info.
+        Plugins with ``redo()`` methods will pick this up.
+
+        **Callbacks**
+
+        Both ``'image-modified'`` and ``'modified'`` callbacks will be issued.
+        To use this from a plugin:
+
+        .. code-block:: python
+
+            image = self.fitsimage.get_image()
+            data = image.get_data()
+            new_data = do_something(data)
+            image.set_data(new_data, metadata=image.metadata)
+            chname = self.fv.get_channelName(self.fitsimage)
+            channel = self.fv.get_channelInfo(chname)
+            channel.image_data_modified()
+
+        Parameters
+        ----------
+        image
+            Image object to update.
+
+        reset : bool
+            Set to `True` on init or if image fell out of cache.
+            This will set the modified timestamp to `None`.
+
+        """
+        if reset:
```

Comment:
@ejeschke , is it worth adding a lock? I don't see a lock in other callback methods. Theoretically, multiple plugins could issue this callback at the same time, but then again, it is quite impossible for the user to use multiple plugins simultaneously, right?